File: Python/Three.py
class Solution:
    def reverse(self, x: int) -> int:
        

        '''

        enter code here
        '''

# Test the solution
solution = Solution()
print(solution.reverse(123))    # Output: 321
print(solution.reverse(-123))   # Output: -321
print(solution.reverse(120))    # Output: 21
print(solution.reverse(1534236469))  # Output: 0 (overflow case)

File: Java/Three.java
package JAVA;

public class Three {
    public int reverse(int x) {
        /*
         Enter code here
         */
        return 0;
    }

    public static void main(String[] args) {
        Three solution = new Three();
        System.out.println(solution.reverse(123));    // Output: 321
        System.out.println(solution.reverse(-123));   // Output: -321
        System.out.println(solution.reverse(120));    // Output: 21
        System.out.println(solution.reverse(1534236469));  // Output: 0 (overflow case)
    }
}
